.jshintrc

```diff
@@ -2,7 +2,7 @@
     "bitwise": true,
     "eqeqeq": true,
     "curly": true,
-    "strict": true,
+    "strict": false,
```

Comment:
@kt3k That makes sense; also, I forgot using ES6 modules implicitly turns "use strict" on, see https://stackoverflow.com/a/31685340/467760